lib/phoenix_live_view/channel.ex

```diff
@@ -561,4 +583,12 @@ defmodule Phoenix.LiveView.Channel do
   defp post_mount_prune(%{socket: socket} = state) do
     %{state | socket: View.post_mount_prune(socket)}
   end
+
+  defp receive_updates(acc) do
+    receive do
+      {@prefix, :send_update, [_ | _] = updates} -> receive_updates(acc ++ updates)
```

Comment:
I am afraid doing this can make we lose causal ordering. For example:

```elixir
send_update(...)
send(self(), ...)
send_update(...)
```

Given we are not updating components in batches (see my comment below), my suggestion is to remove this grouping.